include/pybind11/cast.h

```diff
@@ -754,7 +794,7 @@ template <typename type, typename holder_type> class type_caster_holder : public
     using type_caster_base<type>::value;
     using type_caster_base<type>::temp;
 
-    bool load(handle src, bool convert) {
+    bool load(handle src, bool convert) {//XXX
```

Comment:
(Multiple inheritance involving shared pointer classes TBD, just needs to be copied from above)